### PR TITLE
Display list of active rfps

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
https://olas-website-git-feat-rfp-paths-autonolas.vercel.app/
scroll down to the Launch card, click new "Learn more" button

https://olas-website-git-feat-rfp-paths-autonolas.vercel.app/launch

### CODE REVIEW

File: components/LaunchPage/Opportunities.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      <div className="flex mb-4 gap-4">
+        <Image
+          alt={item.project}
+          src={item.image}
+          width="100"
+          height="100"
+          style={{ objectFit: 'contain' }}
+        />
+        <div>
+          <div className="mb-1 text-sm text-gray-700 font-medium tracking-wider">
```
Comment (truemiller) [inline components/LaunchPage/Opportunities.jsx:28]:
```suggestion
          <span className="block mb-1 text-sm text-gray-700 font-medium tracking-wider">
```

File: components/LaunchPage/Opportunities.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            PROJECT
+          </div>
+          <div className="text-sm font-bold">{item.project}</div>
```
Comment (truemiller) [inline components/LaunchPage/Opportunities.jsx:31]:
```suggestion
          <span className="block text-sm font-bold">{item.project}</span>
```

File: components/LaunchPage/Opportunities.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+      </div>
+      <div className="text-sm mb-4">{item.background}</div>
```
Comment (truemiller) [inline components/LaunchPage/Opportunities.jsx:34]:
```suggestion
      <p className="text-sm mb-4">{item.background}</p>
```

File: components/LaunchPage/Opportunities.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          alt={item.project}
+          src={item.image}
+          width="100"
+          height="100"
+          style={{ objectFit: 'contain' }}
+        />
+        <div>
+          <div className="mb-1 text-sm text-gray-700 font-medium tracking-wider">
+            PROJECT
+          </div>
```
Comment (truemiller) [inline components/LaunchPage/Opportunities.jsx:30]:
```suggestion
          </span>
```

File: components/LaunchPage/Hero.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import SectionHeading from '../SectionHeading';
+import { Button } from '../Button';
+
+const CTA_LINK = 'https://launch.olas.network/';
+
+const Hero = () => (
+  <SectionWrapper customClasses="py-16" backgroundType="SUBTLE_GRADIENT">
+    <div className="grid max-w-screen-xl lg:px-12 mx-auto lg:gap-8 xl:gap-0 lg:grid-cols-12 items-center">
+      <div className="lg:col-span-6 text-center px-5 lg:p-0 lg:text-left mb-12">
+        <div className="mb-6 text-xl tracking-widest uppercase text-slate-700">Olas Launch</div>
```
Comment (truemiller) [inline components/LaunchPage/Hero.jsx:13]:
```suggestion
        <span className="block mb-6 text-xl tracking-widest uppercase text-slate-700">Olas Launch</span>
```

**Review by truemiller — APPROVED** [review]
tiny changes, optional, approved

File: components/LaunchPage/Opportunities.jsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+const CONTACT_URL = 'https://t.me/pahlmeyer';
+
+const OpportunityCard = ({ item }) => (
+  <div className="grid gap-8 md:grid-cols-3 rounded-xl border border-gray-300 shadow-sm p-6 mb-8">
+    <div className="col-span-2 text-start">
+      <h2 className="font-bold text-xl mb-2">{item.agentName}</h2>
+      <pre className="text-l text-gray-600 mb-6 whitespace-pre-line font-sans">
+        {item.agentDescription}
+      </pre>
```
Comment (truemiller) [inline components/LaunchPage/Opportunities.jsx:16]:
```suggestion
      </p>
```

**Review by truemiller — APPROVED** [review]

File: components/LaunchPage/Opportunities.jsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+import { H1 } from 'components/ui/typography';
+import { Button } from 'components/ui/button';
+
+const CONTACT_URL = 'https://t.me/pahlmeyer';
+
+const OpportunityCard = ({ item }) => (
+  <div className="grid gap-8 md:grid-cols-3 rounded-xl border border-gray-300 shadow-sm p-6 mb-8">
+    <div className="col-span-2 text-start">
+      <h2 className="font-bold text-xl mb-2">{item.agentName}</h2>
+      <pre className="text-l text-gray-600 mb-6 whitespace-pre-line font-sans">
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/LaunchPage/Opportunities.jsx]: makes more sense thanks!

File: components/BuildPage/Opportunities.jsx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+const Opportunities = () => (
+  <SectionWrapper customClasses="lg:p-24 px-4 py-12" id="opportunities">
+    <div className="max-w-[1024px] mx-auto mb-12">
+      <H1 className="mb-8 text-center">Opportunities</H1>
+      <p className="text-xl text-gray-600 text-center mb-12">
+        Projects are interested in bringing Olas agents to their ecosystem.
+        This is a list of &quot;requests for agents&quot;
+      </p>
+      {opportunities.map((item) => (
+        <OpportunityCard key={item.agentName} item={item} />
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/BuildPage/Opportunities.jsx:62]: in this case it's fine, but in general I think using spreads providing less control over what is passed to the component, may cause unexpected bugs and re-renders, or some of the fields may be overwritten. e.g.

```
const userData = {
  id: 1,
  name: 'Name',
  lastname: 'Lastname',
  age: 10,
  activity: [{action: 'login', timestamp: '1712904565529'}]
};

const UserSettings = ({ id, name, lastname }) => {
  return (
    <section id={id}>
      <p>Name: {name}</p>
      <p>Lastname: {lastname}</p>
    </section>
  );
};

const App = () => {
  return (
    <div>
      <UserSettings id="userSettings" {...userData} />
    </div>
  );
};
```
Using spread operator here will cause re-render if any field of the itemData is changed even if we don't use them all inside the UserSettings, also the `id` will be overwritten by user's ID

File: components/LaunchPage/Hero.jsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="grid max-w-screen-xl lg:px-12 mx-auto lg:gap-8 xl:gap-0 lg:grid-cols-12 items-center">
+      <div className="lg:col-span-6 text-center px-5 lg:p-0 lg:text-left mb-12">
+        <span className="block mb-6 text-xl tracking-widest uppercase text-slate-700">Olas Launch</span>
+        <SectionHeading size="text-6xl sm:text-7xl lg:text-5xl xl:text-6xl lg:mb-12" color="text-purple-900">
+          Get agents running in your ecosystem
+        </SectionHeading>
+        <Button href={CTA_LINK} isExternal>Get started</Button>
+      </div>
+      <div className="lg:mt-0 lg:col-span-6 lg:flex">
+        <Image src="/images/launch-page/hero.svg" alt="hero" width={834} height={742} className="mx-auto w-3/4 xl:w-full" />
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/LaunchPage/Hero.jsx:20]: you mean width and height? it'll be `w-3/4` (75%) until xl size, looks fine. it's actually copy-pasted from other pages' Heros, I was thinking that I could unite them but hey are too different

File: components/BuildPage/Opportunities.jsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+);
+
+OpportunityCard.propTypes = {
+  item: PropTypes.shape({
+    project: PropTypes.string.isRequired,
+    agentName: PropTypes.string.isRequired,
+    agentDescription: PropTypes.string.isRequired,
+    background: PropTypes.string.isRequired,
+    image: PropTypes.string.isRequired,
+  }).isRequired,
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/BuildPage/Opportunities.jsx]: updated

**Review by truemiller — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**